### PR TITLE
Phone verification

### PR DESCRIPTION
**Pull request by kaisaf**
I added phone verification inside the modal on gear page. Still need to add on the add gear form as well.

### CODE REVIEW

File: gc_project/gc_project/urls.py [hunk 1]
```diff
@@ -23,7 +24,7 @@
                          LoginWidgetView, UserView)
 from gears.views import (HomeView, CategoriesView, CategoryByNameView,
                          GearView, AddGearView, LocationsView,
-                         LocationByNameView)
+                         LocationByNameView, CreateCodeView, ValidateCodeView)
```
Comment (vmenezes) [inline gc_project/gc_project/urls.py:27]: I feel our urls.py is getting ugly with all these imports of individuals views. What you think if we "import gears.views" and just change the urlpatterns list to use views "gear.views.ViewName()"??

File: gc_project/gears/views.py [hunk 2]
```diff
@@ -125,10 +141,7 @@ def get(self, request):
 
     def post(self, request):
         user = User.objects.get(email=request.user.email)
-        if user.phone != request.POST["frmPhone"]:
-            user.phone = request.POST["frmPhone"]
-            user.save()
-
+        phone = request.POST["frmPhone"]
```
Comment (vmenezes) [inline gc_project/gears/views.py:147]: No validation needed in here?

File: gc_project/gc_project/urls.py (hunk 1 above)
Comment (kaisaf) [inline gc_project/gc_project/urls.py:27]: Agreed, will change.

File: gc_project/gears/views.py (hunk 2 above)
Comment (kaisaf) [inline gc_project/gears/views.py:147]: Yes, I mentioned this in my note above ("Still need to add on the add gear form as well."). Not sure what I was thinking whenever those lines 128-131 were removed. I will take care of this too.

**Comment by kaisaf** [comment]
I added phone verification on the addgear form. I hid the rest of the form while waiting to get the correct pin. I also did an overall code cleanup. However, I didn't change the urls.py, because there are different sets of views that are imported from different apps. Wanted to keep it consistent as of now and maybe discuss.

**Pull request merged**